packages/create-cloudflare/src/cli.ts

```diff
@@ -201,4 +201,6 @@ main(process.argv)
 	})
 	.finally(async () => {
 		await reporter.waitForAllEventsSettled();
+		// await setTimeout(100);
+		process.exit();
```

Comment:
my question here is ... do we **always** want to exit the process? or should we restrict this to sigint and sigterm signals only?

Reply:
I think it is totally fine to call `process.exit()` all the time. We were calling it everywhere before we added telemetry... 😅

Reply:
Wrangler is doing it all the time: https://github.com/cloudflare/workers-sdk/blob/47f56a2a4dcfa216d768fec974c2fa8c5bb3c21c/packages/wrangler/src/cli.ts#L20-L26